distributed/tests/test_client_executor.py

```diff
@@ -116,7 +107,7 @@ def test_cancellation(client):
 
 def test_cancellation_wait(client):
     with client.get_executor(pure=False) as e:
-        fs = [e.submit(slowinc, i, delay=0.1) for i in range(10)]
+        fs = [e.submit(slowinc, i, delay=0.2) for i in range(10)]
```

Comment:
test was randomly failing as the cancellation occasionally took more than 0.1s to be enacted and thus one of the tasks had enough time to finish